src/presentation/api/controller/sslController.go

```diff
@@ -125,3 +126,26 @@ func DeleteSslPairController(c echo.Context) error {
 
 	return apiHelper.ResponseWrapper(c, http.StatusOK, "SslPairDeleted")
 }
+
+func SslCertificateWatchDogController() {
+	validationIntervalMinutes := 60 / useCase.SslValidationsPerHour
+
+	taskInterval := time.Duration(validationIntervalMinutes) * time.Minute
+	timer := time.NewTicker(taskInterval)
+	defer timer.Stop()
+
+	sslQueryRepo := sslInfra.SslQueryRepo{}
+	sslCmdRepo := sslInfra.NewSslCmdRepo()
+	vhostQueryRepo := vhostInfra.VirtualHostQueryRepo{}
+	vhostCmdRepo := vhostInfra.VirtualHostCmdRepo{}
+
+	for range timer.C {
+		sslCertificateWatchDog := useCase.NewSslCertificateWatchDog(
+			sslQueryRepo,
+			sslCmdRepo,
+			vhostQueryRepo,
+			vhostCmdRepo,
+		)
+		sslCertificateWatchDog.Execute()
```

Comment:
Seria interessante se o seu Execute retornasse errors e você pudesse logar eles num único local (aqui), não?

Reply:
Conforme conversamos, está sendo executando um for loop lá dentro, então é melhor manter os logs lá.